src/routes/home/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
+import { Card, Button, Alert, Table } from "react-bootstrap";
+import { useAuth } from "../../components/context";
+import { Link, useHistory } from "react-router-dom";
+import app from "../../components/Firebase/firebase";
+
+export default function Home() {
+  const [error, setError] = useState("");
+  const { logout } = useAuth();
+  const [complains, setComplains] = useState({});
+  const history = useHistory();
+
+  useEffect(() => {
+    const todoRef = app.database().ref("complains");
+    todoRef.on("value", (snapshot) => {
+      const todos = snapshot.val();
+      const todoList = [];
+      for (let id in todos) {
+        todoList.push({ id, ...todos[id] });
+      }
+      setComplains(todoList);
+    });
+  }, []);
+
+  async function handleLogout() {
+    setError("");
+
+    try {
+      await logout();
+      history.push("/login");
+    } catch {
+      setError("Failed to log out");
+    }
+  }
+  return (
+    <>
+      <Card>
+        <Card.Body>
+          <h2 className="text-center mb-4">Home</h2>
+          {error && <Alert variant="danger">{error}</Alert>}
+        </Card.Body>
+      </Card>
+      <Table striped bordered hover variant="light">
```

Comment:
Very much hard coded UI, try to implement a table component which will render according to data.